### DISCUSSION TITLE
Gaussian Mixture Model

### DISCUSSION THREAD

**Discussion by vaidishs** [mir-group/allegro#37] [Q&A]
There is a development branch for GMM models in the nequip git, is there a plan to add GMM to allegro too?

**Comment by Linux-cpp-lisp** [comment]
Hi @vaidishs ,

Thanks for your interest in our methods!

Yes, we've just merged support for our GMM UQ method for NequIP on the `develop` branch of `nequip`, but such a method is not yet available for Allegro.  We'll be sure to post here about any future developments for UQ with Allegro.
